src/utils/input-validators.js

```diff
@@ -5,6 +5,21 @@ function isValidPassword({ password, minLength = 8 }) {
   return sanitizedPassword.length >= minLength && minOneLetterAndOneDigitPattern.test(sanitizedPassword);
 }
 
+function isValidTag(allTags, tag) {
+  const trimmedTag = (tag || '').trim();
+
+  if (trimmedTag.length < 3 || trimmedTag.length > 25 || trimmedTag.includes(' ')) {
+    return false;
+  }
+
+  if (allTags.filter(t => t === trimmedTag).length > 1) {
```

Comment:
Yeah that would be much nicer if we were comparing `length > 0` but since we invalidate for more than 1, we can't use `some` as it will return `true` for 1 match.

Reply:
Didn't see you were searching for greater than 1, I thought we were avoiding multiple occurrences of the same tag. I guess it doesn't something else.